### DISCUSSION TITLE
how to tap a flutter checkbox

### DISCUSSION THREAD

**Discussion by maxcaetanoo** [mobile-dev-inc/Maestro#1571] [Q&A]
I try to select a checkbox but I can't find a solution, selecting by ID doesn't work and I don't have a toll tip or key in the checkbox, please help me😫

How can I do this with the Maestro

**Comment by imdishasharma** [comment]
I was also facing the same issue but you can tapOn the near by element then the check box will be selected ! that works for me hope it works for you as well! btw it's too late though
